core/src/main/java/org/radargun/stages/test/Stressor.java

```diff
@@ -68,7 +68,7 @@ public Stressor(TestStage stage, OperationLogic logic, int globalThreadIndex, in
    }
 
    private boolean recording() {
-      return this.started && !stage.isFinished();
+      return this.started;
```

Comment:
@diegolovison curious why you changed this line?

Reply:
Because it will generate different numbers of requests.
The fix is the line that you mentioned and https://github.com/radargun/radargun/pull/635/files#diff-9158f9969a715eda006055cd03e939d4R147

For example, a request could be reported in statistics in https://github.com/radargun/radargun/pull/635/files#diff-9158f9969a715eda006055cd03e939d4R189 but not in https://github.com/radargun/radargun/pull/635/files#diff-9158f9969a715eda006055cd03e939d4R251